athena-hbase/src/main/java/com/amazonaws/athena/connectors/hbase/HbaseMetadataHandler.java

```diff
@@ -211,8 +219,23 @@ public GetTableResponse doGetTable(BlockAllocator blockAllocator, GetTableReques
                     ex);
         }
 
+        String schemaName;
+        String tableName;
+
+        if (request.isQueryPassthrough()) {
```

Comment:
actually can we not forward the call `doGetQueryPassthroughSchema` to `doGetTable` 

Please keep them separated; and if there is a common logic, make it into common private method.

Reply:
Hi @AbdulR3hman Refactored code.